packages/client/hmi-client/src/components/dataset/tera-dataset.vue

```diff
@@ -256,14 +256,18 @@
 								<span>Suggested value</span>
 								<div>
 									<div class="suggested-value-source">
-										<i class="pi pi-file" />{{ dataset.metadata?.documents[0].title }}
+										<i class="pi pi-file" />{{
+											dataset.metadata?.documents ? dataset.metadata?.documents[0].title : ''
```

Comment:
Nitpick: you can make it shorter - `dataset.metadata?.documents?.[0].title ?? ''`